public/v4/apps/codimd.yml

```diff
@@ -0,0 +1,43 @@
+captainVersion: 4
+services:
+    $$cap_appname-mariadb:
+        image: linuxserver/mariadb:110.4.14mariabionic-ls77
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_DB_PASS
+            MYSQL_DATABASE: codimd
+            MYSQL_USER: codimd
+            MYSQL_PASSWORD: $$cap_DB_PASS
+            PGID: 1000
+            PUID: 1000
+            TZ: Europe/London
+        volumes:
+            - $$cap_appname-mariadb-config:/config
+    $$cap_appname:
+        environment:
+            DB_HOST: srv-captain--$$cap_appname-mariadb
+            DB_USER: codimd
+            DB_PASS: $$cap_DB_PASS
+            DB_NAME: codimd
+            DB_PORT: 3306
+            PGID: 1000
+            PUID: 1000
+            TZ: Europe/London
+        image: linuxserver/codimd:1.6.0-ls44
+        restart: always
+        volumes:
+            - $$cap_appname-config:/config
+        caproverExtra:
+            containerHttpPort: '3000'
+caproverOneClickApp:
+    variables:
+        - label: Database Password
+          description: Password to connect to the database
+          defaultValue: daTaBaSePaSsWorD
+          id: $$cap_DB_PASS
+    instructions:
+        start: Realtime collaborative markdown notes on all platforms
```

Comment:
Please add more details here. Perhaps a link to github or the official website.

Reply:
Answered in 3adeac0, is that better? 😄